modules/cli/src/test/java/org/apache/ignite/cli/deprecated/IgniteCliInterfaceTest.java

```diff
@@ -445,58 +404,27 @@ void updateHocon() {
     @Nested
     @DisplayName("cluster")
     class Cluster {
-        @Mock
-        private HttpClient httpClient;
-
-        @Mock
-        private HttpResponse<String> response;
-
-        private ClusterConfigShowCallInput capturedShowConfigurationInput;
-
-        private ClusterConfigUpdateCallInput capturedUpdateConfigurationInput;
-
-        ClusterConfigShowCall showConfigurationCall() {
-            return new ClusterConfigShowCall() {
-                @Override
-                public DefaultCallOutput<String> execute(ClusterConfigShowCallInput input) {
-                    capturedShowConfigurationInput = input;
-                    return DefaultCallOutput.success("{\"autoAdjust\":{\"enabled\":true}}");
-                }
-            };
-        }
-
-        ClusterConfigUpdateCall updateConfigurationCall() {
-            return new ClusterConfigUpdateCall() {
-                @Override
-                public DefaultCallOutput<String> execute(ClusterConfigUpdateCallInput input) {
-                    capturedUpdateConfigurationInput = input;
-                    return DefaultCallOutput.success("Configuration was updated successfully.");
-                }
-            };
-        }
-
-        @BeforeEach
-        void setUp() {
-            ctx.registerSingleton(httpClient);
-            ctx.registerSingleton(showConfigurationCall());
-            ctx.registerSingleton(updateConfigurationCall());
-        }
-
         @Test
-        @DisplayName("init --node-endpoint=127.0.0.1:17300 --meta-storage-node node1ConsistentId --meta-storage-node node2ConsistentId "
-                + "--cmg-node node2ConsistentId --cmg-node node3ConsistentId")
-        void initSuccess() throws Exception {
-            when(response.statusCode()).thenReturn(HttpURLConnection.HTTP_OK);
-            when(httpClient.<String>send(any(), any())).thenReturn(response);
-
+        @DisplayName("init --cluster-url http://localhost:10300 --meta-storage-node node1ConsistentId --meta-storage-node node2ConsistentId "
+                + "--cmg-node node2ConsistentId --cmg-node node3ConsistentId --cluster-name cluster")
+        void initSuccess() {
             var expSentContent = "{\"metaStorageNodes\":[\"node1ConsistentId\",\"node2ConsistentId\"],"
```

Comment:
exp means expression, exponent, expected? I think expected prefix is ok and there is no need to cut.